app/components/concept-scheme-uri-selector.js

```diff
@@ -42,20 +42,15 @@ export default class ConceptSchemeUriSelectorComponent extends Component {
     this.update();
   }
 
-  async queryDB(query) {
-    const encodedQuery = escape(query);
-    const endpoint = `${SPARQL_ENDPOINT}?query=${encodedQuery}`;
-    const response = await fetch(endpoint, {
-      headers: { Accept: 'application/sparql-results+json' },
+  getSortedOptions(conceptSchememodels) {
+    return [...conceptSchememodels].sort(function (a, b) {
+      if (a.label < b.label) {
+        return -1;
+      }
+      if (a.label > b.label) {
+        return 1;
+      }
+      return 0;
```

Comment:
Can this be fixed by adding a sort on label in the store query?

Reply:
I tried it with the `sort: 'preflabel`' in the query but this is resulting in inconsisten results IMO 
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/cd93d248-5500-4c64-baa5-299adcb3130e)